lms/djangoapps/grades/tasks.py

```diff
@@ -271,7 +271,6 @@ def _has_db_updated_with_new_score(self, scored_block_usage_key, **kwargs):
                 "student_id": kwargs['anonymous_user_id'],
                 "course_id": str(scored_block_usage_key.course_key),
                 "item_id": str(scored_block_usage_key),
-                "item_type": scored_block_usage_key.block_type,
```

Comment:
Did you talk to dave about this?

Reply:
I posted the question on TNL

Reply:
@leangseu-edx: Yeah, this should be fine. The unique index use for lookup doesn't rely on `item_type` at all: https://github.com/edx/edx-submissions/blob/131d4f3cd2ba2fb6ab78ecb3a19b32357054fbc5/submissions/models.py#L95-L98